packages/contracts-bedrock/test/dispute/SuperFaultDisputeGame.t.sol

```diff
@@ -2474,38 +2310,6 @@ contract SuperFaultDisputeGame_Test is SuperFaultDisputeGame_Init {
         assertEq(uint8(gameProxy.bondDistributionMode()), uint8(BondDistributionMode.NORMAL));
     }
 
-    /// @dev Tests that closeGame called with any amount of gas either reverts (with OOG) or
-    ///      updates the anchor state. This is specifically to verify that the try/catch inside
-    ///      closeGame can't be called with just enough gas to OOG when calling the
-    ///      AnchorStateRegistry but successfully execute the remainder of the function.
-    /// @param _gas Amount of gas to provide to closeGame.
-    function testFuzz_closeGame_canUpdateAnchorStateAndDoes_succeeds(uint256 _gas) public {
```

Comment:
This test doesn't rely on the AlphabetVM. We don't need to delete it and I'd be surprised if this isn't passing.